README.md

```diff
@@ -112,3 +112,7 @@ Run the following commands:
 docker build --target test --tag todo-app:test .
 docker run todo-app:test
 ```
+
+## Deployement link
+
+https://aqsatodo.azurewebsites.net/
```

Comment:
Technically this isn't enough to cover the submission criteria here:
![image](https://github.com/user-attachments/assets/78d3a1e1-0b0e-4c56-94a3-b382a507c75c)